techlibs/quicklogic/synth_quicklogic.cc

```diff
@@ -173,7 +173,7 @@ struct SynthQuickLogicPass : public ScriptPass {
         if (check_label("map_gates")) {
             if (inferAdder && family != "pp3")
             {
-                run("ap3_wrapcarry");
+                run("quicklogic_fixcarry");
```

Comment:
Could you please explain what is the intention of the `quicklogic_fixcarry` pass? What does it do?

Reply:
Not required. I commented it now as ap3_arith_map.v does mapping.